serverless.ts

```diff
@@ -338,6 +338,44 @@ const serverlessConfig: AWS = {
         ALGOLIA_INDEX: `asap-hub_${envRef}`,
       },
     },
+    labUpserted: {
+      handler: 'apps/asap-server/src/handlers/webhooks/webhook-lab.handler',
+      events: [
+        {
+          httpApi: {
+            method: 'POST',
+            path: '/webhook/labs',
+          },
+        },
+      ],
+      environment: {
+        EVENT_BUS: 'asap-events-${self:provider.stage}',
+        EVENT_SOURCE: 'asap.lab',
+      },
+    },
+    indexLabUsers: {
+      handler: 'apps/asap-server/src/handlers/lab/users-handler.handler',
+      events: [
+        {
+          eventBridge: {
+            eventBus: 'asap-events-${self:provider.stage}',
+            pattern: {
+              source: ['asap.lab'],
+              'detail-type': [
+                'LabPublished',
+                'LabUpdated',
+                'LabCreated',
```

Comment:
i don't think we need to listen to `LabCreated` event, what do you think? since there should be no users to update anyway

Reply:
Good point ;)
However - we can get this event anytime - maybe 1h after creation due to network conditions.